### DISCUSSION TITLE
关于项目周期

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#197] [Q&A] [answered]
请问项目是不是要求在阶段3就必须完成所有基础功能和pdf中给出的可选功能？
也就是说，阶段3（也就是项目开始后6个礼拜）如果没有完成前阶段给出的可选功能，就拿不到可选功能对应的分数了？

**Comment by liusy58** [comment, marked as answer]
并非如此，但是**可能**会有一些些折扣
